src/UglyToad.PdfPig/Geometry/GeometryExtensions.cs

```diff
@@ -54,7 +67,7 @@ public static PdfPoint Subtract(this PdfPoint point1, PdfPoint point2)
         /// </param>
         internal static PdfRectangle ParametricPerpendicularProjection(IReadOnlyList<PdfPoint> polygon)
         {
-            if (polygon == null || polygon.Count == 0)
+            if (polygon == null || !polygon.Any())
```

Comment:
Just as a general performance pointer, I'd avoid LINQ wherever you can, while it's more readable in this case it's very easy to end up accidentally putting LINQ on a hot path and clobbering perf. For instance I wrote a quick timing test for `Any` versus `Count` including lists with 0 and 1 items. The stats for 700 runs each looped 100,000 times were:

```
Count: Min: 3618        Max: 36413      Avg: 8659.79
  Any: Min: 15632       Max: 64904      Avg: 23914.74
```
Where the values are in ticks. It's possible I made an error writing this test and should have just used benchmark dot net but I think in the general case you're going to see worse performance with LINQ vs the 'naive' code alternative wherever it is used.

Now it doesn't matter too much here since this isn't a hot path as far as I'm aware but avoiding linq is a habit I like to keep to.